kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -62,6 +62,7 @@ function _contentState(data) {
     available: data.available,
     files: data.files,
     progress,
+    breadcrumbs: [],
```

Comment:
possible a line of code was in accidentally deleted?

https://github.com/learningequality/kolibri/blob/4aac23f37f182f289d24313da02c40c9cba32058/kolibri/plugins/learn/assets/src/state/actions.js#L66